.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
Should we use some specific version instead of `latest` for stability?

Reply:
`latest` tag is for the latest Ubuntu LTS version but i agree with you, i will set it to  `ubuntu-20.04`